src/config/config.go

```diff
@@ -10,6 +10,7 @@ type ArgumentList struct {
 	DockerClientVersion   string `default:"1.24" help:"Optional. Specify the version of the docker client. Used for compatibility."`
 	DisableStorageMetrics bool   `default:"false" help:"Disables storage driver metrics collection."`
 	ShowVersion           bool   `default:"false" help:"Print build information and exit"`
+	UseDockerAPI          bool   `default:"false" help:"Leverage Docker API to gather metrics instead of filesystem. This option is ignored if cgroupsV1 are detected"`
```

Comment:
just a nit but i would move this and the `Fargate` arg together and use a similar description , so is clear that the both flags change the source of the metrics. 

Ideally they should have a similar name like `UseFargate` and `UseDockerAPI` but that is not a nice breaking change.